src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -638,6 +642,25 @@ bool CommandRequestImpl::CheckHMICapabilities(
   return false;
 }
 
+std::string CommandRequestImpl::GetPermitResultInfoMessage() const {
```

Comment:
@AKalinich-Luxoft 
I think it's better to convert it to free function and just pass result as parameter.

Reply:
@dev-gh done in https://github.com/smartdevicelink/sdl_core/pull/1799/commits/a693891743df516a77f3a65c451f24b5da9aafac